### ISSUE TITLE
QR code will bring user to create job page for that service provider

### ISSUE THREAD

**Issue by mahcloud** [WhosAble/WhosAble#65]
![img_9087](https://cloud.githubusercontent.com/assets/1529103/15836164/bb1e3498-2bf0-11e6-927c-851ea7e72f1b.PNG)

**Comment by dooliddler** [comment]
Most QR code generators have a feature that allows you to create a contact card QR code. The first one that I checked out have ability to capture first name last name email address and URL for a website and a phone number. Ideally we would also have the ability to specify a service type.

**Comment by mahcloud** [comment]
@dooliddler I would imagine we would actually use the web url version.
http://goqr.me/api/ seems like it is free and easy to use. Thoughts?

**Comment by mahcloud** [comment]
something like this https://api.qrserver.com/v1/create-qr-code/?size=150x150&data=http://www.whosable.com/qr_loader?id=1

**Comment by dooliddler** [comment]
Yes.  Did you scan the image I texted to you?

**Comment by mahcloud** [comment]
@dooliddler Yes, I did scan the qr code from this business card, but that just opens a note.
The qrserver.com that I suggested creates a code like 
![image](https://cloud.githubusercontent.com/assets/1529103/15842346/74d94b7e-2c15-11e6-98de-d3e2f3040e6c.png) that will open a url in a browser.

**Comment by dooliddler** [comment]
Perfect.  Do you think it Possible to launch directly into the create a job form with name, service type and phone populated?  I would imagine the SP would already have to have created an account

**Comment by mahcloud** [comment]
@dooliddler yes, I think we can make the url that qr code goes to do exactly that.
